### PR TITLE
adjusting variables to match changed module requirements

### PR DESCRIPTION
**Pull request by PetrMc**
per AWS validattion:
```
I am getting the following output when running terraform plan (I assume something basic in the [main.tf](https://www.google.com/url?q=http://main.tf&sa=D&source=docs&ust=1657747791525554&usg=AOvVaw2arOiUofp48jYY-uyRJ1Ml)):

│ Error: Unsupported argument
│
│ on [main.tf](https://www.google.com/url?q=http://main.tf&sa=D&source=docs&ust=1657747791525655&usg=AOvVaw2fUvWjjqJmY7C2oOTTIxkL) line 103, in module "aws_eks_accelerator_for_terraform":
│ 103: tenant = local.tenant
│
│ An argument named "tenant" is not expected here.
╵
╷
│ Error: Unsupported argument
│
│ on [main.tf](https://www.google.com/url?q=http://main.tf&sa=D&source=docs&ust=1657747791525705&usg=AOvVaw0lb7TElHMQ5BaxTb6c51Ld) line 104, in module "aws_eks_accelerator_for_terraform":
│ 104: environment = local.environment
│
│ An argument named "environment" is not expected here.
╵
╷
│ Error: Unsupported argument
│
│ on [main.tf](https://www.google.com/url?q=http://main.tf&sa=D&source=docs&ust=1657747791525751&usg=AOvVaw1up_-jQnFcBkK-v2O2DgK8) line 105, in module "aws_eks_accelerator_for_terraform":
│ 105: zone = local.zone
│
│ An argument named "zone" is not expected here.
╵
╷
│ Error: Unsupported argument
│
│ on [main.tf](https://www.google.com/url?q=http://main.tf&sa=D&source=docs&ust=1657747791525805&usg=AOvVaw3XBULPBlg0L--PNUIvDCfp) line 106, in module "aws_eks_accelerator_for_terraform":
│ 106: terraform_version = local.terraform_version
│
│ An argument named "terraform_version" is not expected here.

TF version:
on darwin_amd64
+ provider [registry.terraform.io/gavinbunney/kubectl](https://www.google.com/url?q=http://registry.terraform.io/gavinbunney/kubectl&sa=D&source=docs&ust=1657747791525854&usg=AOvVaw1LKX950KDpbK10UnvLvPzZ) v1.14.0
+ provider [registry.terraform.io/hashicorp/aws](https://www.google.com/url?q=http://registry.terraform.io/hashicorp/aws&sa=D&source=docs&ust=1657747791525892&usg=AOvVaw0S3zlzWoypEY0GIzfZKKQH) v4.22.0
+ provider [registry.terraform.io/hashicorp/cloudinit](https://www.google.com/url?q=http://registry.terraform.io/hashicorp/cloudinit&sa=D&source=docs&ust=1657747791525929&usg=AOvVaw3Fl2xo3DxO3j1NuebJw9mR) v2.2.0
+ provider [registry.terraform.io/hashicorp/helm](https://www.google.com/url?q=http://registry.terraform.io/hashicorp/helm&sa=D&source=docs&ust=1657747791525964&usg=AOvVaw36hSwalJNkS9atrbRLEDir) v2.6.0
+ provider [registry.terraform.io/hashicorp/kubernetes](https://www.google.com/url?q=http://registry.terraform.io/hashicorp/kubernetes&sa=D&source=docs&ust=1657747791526000&usg=AOvVaw2gsCNNaZG678b4prmd7Qyr) v2.12.1
+ provider [registry.terraform.io/hashicorp/local](https://www.google.com/url?q=http://registry.terraform.io/hashicorp/local&sa=D&source=docs&ust=1657747791526040&usg=AOvVaw2vVuPB0va1Fvfe-mPwZ7R9) v2.2.3
+ provider [registry.terraform.io/hashicorp/null](https://www.google.com/url?q=http://registry.terraform.io/hashicorp/null&sa=D&source=docs&ust=1657747791526079&usg=AOvVaw3xT16MYWxJ_Qc30-Cdojn1) v3.1.1
+ provider [registry.terraform.io/hashicorp/time](https://www.google.com/url?q=http://registry.terraform.io/hashicorp/time&sa=D&source=docs&ust=1657747791526114&usg=AOvVaw0hOC8jZ9w8lKGsLM_itazZ) v0.7.2
+ provider [registry.terraform.io/hashicorp/tls](https://www.google.com/url?q=http://registry.terraform.io/hashicorp/tls&sa=D&source=docs&ust=1657747791526148&usg=AOvVaw183Odr6KM2igXVbl0g4q_Z) v3.4.0
+ provider [registry.terraform.io/terraform-aws-modules/http](https://www.google.com/url?q=http://registry.terraform.io/terraform-aws-modules/http&sa=D&source=docs&ust=1657747791526183&usg=AOvVaw1FoR_dxXHmsMwJIBa8eqT-) v2.4.1
Anonymous
```

those variables caused the problem: 
```
  # tenant            = local.tenant
  # environment       = local.environment
  # zone              = local.zone
  # terraform_version = local.terraform_version```
```
this varialbe was "missing":
```
  cluster_name = local.cluster_name
```

### CODE REVIEW

**Review by shapirov103 — COMMENTED** [review]
Looks good, let's merge it and I will test it out. All issues appear to be addressed.

File: blueprints/getting-started/main.tf [hunk 1]
```diff
@@ -100,10 +102,11 @@ module "aws_vpc" {
...
   source = "github.com/aws-ia/terraform-aws-eks-blueprints"
 
-  tenant            = local.tenant
-  environment       = local.environment
-  zone              = local.zone
-  terraform_version = local.terraform_version
+  # tenant            = local.tenant
+  # environment       = local.environment
+  # zone              = local.zone
+  # terraform_version = local.terraform_version
```
Comment (yskopets) [inline blueprints/getting-started/main.tf]: These lines should be removed rather than commented

**Review by yskopets — APPROVED** [review]
I see that upstream [example](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/examples/eks-cluster-with-new-vpc/) has changed significantly.

I would suggest to copy the entire new example.

There were only 2 lines in this example different from upstream:

```tf
  # Tetrate Istio Add-on
  enable_tetrate_istio = true
```

**Comment by PetrMc** [comment]
thanks @shapirov103 and @yskopets for the comments and review.

the new commit takes @yskopets recommendation to pull the new upstream example and re-apply changes.

@shapirov103 can you please kindly retest and let me know if it works in your environment. (my tests are in progress)

**Comment by shapirov103** [comment]
> thanks @shapirov103 and @yskopets for the comments and review.
> 
> the new commit takes @yskopets recommendation to pull the new upstream example and re-apply changes.
> 
> @shapirov103 can you please kindly retest and let me know if it works in your environment. (my tests are in progress)

validated, works fine

**Comment by PetrMc** [comment]
@shapirov103 thanks a lot for the validation! we merged to the master. Please let us know of any other outstanding items

**Pull request merged**